applications/tari_dan_app_utilities/src/consensus_constants.rs

```diff
@@ -35,6 +36,7 @@ impl ConsensusConstants {
             committee_size: 7,
             max_base_layer_blocks_ahead: 5,
             max_base_layer_blocks_behind: 5,
+            max_block_time_threshold: 10,
```

Comment:
Can make this a `Duration` type
I like `pacemaker_max_base_block_time` as the name, but pacemaker prefix can be left off if. `_base_` is there to clarify that the actual block time/leader timeout is always more than this. But open to suggestions or leaving it off.
```suggestion
            pacemaker_max_base_block_time: Duration, 
```

Reply:
Maybe `pacemaker` prefix is a good thing to help clarify where this time goes? `base` also makes sense this this is not the actual threshold, good spot.